hooks/pre/10-reset_feature.rb

```diff
@@ -57,4 +63,11 @@ def reset_pulp
   Kafo::Helpers.execute(commands)
 end
 
-reset if app_value(:reset) && !app_value(:noop)
+if app_value(:reset) && !app_value(:noop)
+  response = ask('Are you sure you want to continue? [y/n]')
+  if response == 'n' || response == 'N'
+    $stderr.puts '** cancelled **'
+    kafo.class.exit(1)
+  elsif reset
```

Comment:
But, the logic should be reversed, only 'Y/y' should trigger the reset.  The way it's written now if a user types anything but N or n, it would reset